crates/goose-cli/src/session/output.rs

```diff
@@ -465,14 +477,18 @@ pub fn env_no_color() -> bool {
 }
 
 fn print_markdown(content: &str, theme: Theme) {
-    bat::PrettyPrinter::new()
-        .input(bat::Input::from_bytes(content.as_bytes()))
-        .theme(theme.as_str())
-        .colored_output(env_no_color())
-        .language("Markdown")
-        .wrapping_mode(WrappingMode::NoWrapping(true))
-        .print()
-        .unwrap();
+    if std::io::stdout().is_terminal() {
+        bat::PrettyPrinter::new()
```

Comment:
I am less familiar with `bat` and this `cliclack` crate but IME with https://docs.rs/indicatif/latest/indicatif/ it's pretty easy to use e.g. https://docs.rs/indicatif/latest/indicatif/struct.ProgressBar.html#method.println to automate these kinds of conditional prints.

Reply:
looks cool. here we are just skipping the formatting of markdown though